package.json

```diff
@@ -5,10 +5,9 @@
 	"main": "server.js",
 	"type": "module",
 	"scripts": {
-		"start": "nodemon src/server.js",
-		"test": "echo \"Error: no test specified\" && exit 1"
+		"start": "nodemon src/server.js"
 	},
-	"author": "Ram Manthry",
+	"author": "",
 	"license": "ISC",
 	"dependencies": {
 		"dotenv": "^16.0.3",
```

Comment:
Do we better install the typescript and Jest, and set up the jest.config.js in this base api first?

Reply:
Done